src/gax/src/error/core_error.rs

```diff
@@ -119,6 +119,65 @@ impl Error {
         matches!(self.kind, ErrorKind::Timeout)
     }
 
+    /// Not part of the public API, subject to change without notice.
+    ///
+    /// Creates an error representing a deserialization problem.
+    ///
+    /// Applications should have no need to use this function. The exception
+    /// could be mocks, but this error is too rare to merit mocks. If you are
+    /// writing a mock that extracts values from [wkt::Any], consider using a
+    /// `.expect()` calls instead.
+    ///
+    ///
+    /// # Example
+    /// ```
+    /// use std::error::Error as _;
+    /// use google_cloud_gax::error::Error;
+    /// let error = Error::deser("simulated problem");
+    /// assert!(error.is_deserialization());
+    /// assert!(error.source().is_some());
+    /// ```
+    #[cfg_attr(not(feature = "_internal-semver"), doc(hidden))]
+    pub fn deser<T: Into<BoxError>>(source: T) -> Self {
+        Self {
+            kind: ErrorKind::Deserialization,
+            source: Some(source.into()),
+        }
+    }
+
+    /// The request could not be completed before its deadline.
```

Comment:
"The response could not be deserialized."

Reply:
LOL, now you know where I cut&pasted this from.